frontend/src/app/[locale]/search/page.tsx

```diff
@@ -71,6 +72,35 @@ function Search({ searchParams }: { searchParams: searchParamsTypes }) {
   const pager1key = Object.entries(searchParams).join("-") + "pager1";
   const pager2key = Object.entries(searchParams).join("-") + "pager2";
 
+  const filterConfigurations = useMemo((): SearchFilterConfiguration[] => {
```

Comment:
refactored to dry up the accordion implementation and hopefully improve extensibility if we need it

Reply:
Don't feel strongly, but this feels less extensible and could make it harder to change in the future, ie if another component type is added among the filters, but uses the same amount of code.

Reply:
Impartial, although given what we were talking about this morning with how next caches data fetches, I wonder if we could make a data fetch call inside SearchFilters with no performance cost as that same call would also be made in this component, this could yield to passing less things in?

Reply:
I think a better way to do it might be to move any functionality related to the search params out of `Search` and into lower level components, and just take minor hit on processing the params multiple times. That would be slightly bigger refactor with a wider surface area though.

For now, I could just move the search param logic for the filter accordions into SearchFilters, and within that file keep things looking more like they do now? Or I could just pass through the necessary param related info rather than the config idea. Open to whatever.

Reply:
pushed a commit with another approach, and the only thing I don't like is the sheer number of props that search filters now needs to take (9). If we're ok with that then this could work? Have a couple of ideas percolating, but this client / server boundary stuff is adding a wrench to my usual thinking on this stuff